runtime/web_worker.rs

```diff
@@ -392,6 +393,7 @@ impl WebWorker {
         file_fetch_handler: Rc::new(deno_fetch::FsFetchHandler),
         ..Default::default()
       }),
+      deno_cache::init::<SqliteBackedCache>(None).disable(),
```

Comment:
Why add it if it's always disabled?

Reply:
Cache API is supported in WebWorker context, I've enabled it.